test/snapshot.test.js

```diff
@@ -92,3 +94,32 @@ test('snapshot reader favours bundled snapshot when possible', async (t) => {
   existsStub.restore();
   t.end();
 });
+
+test('reader loading snapshot from upstream', async (t) => {
+  nock('https://homebase.snyk.io')
```

Comment:
is it supposed to be repeated twice?

Reply:
yes, this tells `nock` to expect two requests (see our e2e test for more reference)